webserver.md

```diff
@@ -0,0 +1,280 @@
+{::options parse_block_html="true" /}
+
+<div class="row">
+
+<div class="large-12 columns">
+# Build Your Own Internet
+
```

Comment:
I definitely see how it will be cleaner. The alternative way to connect to the Tessel server is through its IP Address while sharing a local network. I think the steps for describing that process could make the tutorial a bit more complicated than creating an access point. 

We could describe that process of connecting over a shared network in a bonus or note at the bottom, so the readers know all the ways they can connect to Tessel.